datatypes/src/primitives/bounding_box.rs

```diff
@@ -386,6 +386,39 @@ impl BoundingBox2D {
             None
         }
     }
+
+    pub fn add_coord(&mut self, coord: Coordinate2D) {
+        self.lower_left_coordinate = self.lower_left_coordinate.min_elements(coord);
+        self.upper_right_coordinate = self.upper_right_coordinate.max_elements(coord);
+    }
```

Comment:
I'm not quite sure if you want to add points to a bounding box, because they are gone after the operation.

Maybe just calculate min/max in `from_coord_iter`?

Reply:
i renamed it to extend_with_coord.